outputs.tf

```diff
@@ -28,31 +38,21 @@ output "ingress" {
   }
 }
 
-output "product_urls" {
-  description = "URLs to access the deployed Atlassian products"
-
-  value = {
-    bamboo = module.bamboo.product_domain_name
-  }
-}
-
-output "database" {
+output "bamboo-database" {
   description = "Database information"
 
-  value = {
+  value = local.install_bamboo ? {
     rds_instance_id        = module.bamboo.rds_instance_id
     db_name                = module.bamboo.db_name
     kubernetes_secret_name = module.bamboo.kubernetes_rds_secret_name
     jdbc_connection        = module.bamboo.rds_jdbc_connection
-  }
+  } : null
```

Comment:
Will this mean the value won't be included in the output at all?

Reply:
In case of `install_bamboo` is `false` it will print output like this:
```
bamboo_database: null
```

But prevent the error in installation, because `module.bamboo` does not existed.